app/services/loader.js

```diff
@@ -13,7 +13,7 @@ export default Service.extend({
     if (config.isExternal) {
       options.url = url;
     } else {
-      options.url = config.withoutPrefix ? `${adapter.host}/${url}` : `${adapter.urlPrefix()}${url}`;
+      options.url = config.withoutPrefix ? `${adapter.host}${url}` : `${adapter.urlPrefix()}${url}`;
```

Comment:
@sumedh123 forget withoutPrefix for a second. Irrespective of if it is true or false, add a leading slash to the `url` value if it doesn't have one already.